src/panoptes/utils/time.py

```diff
@@ -212,11 +215,11 @@ def sleep(self, max_sleep: Union[int, float, None] = None, log_level: str = "DEB
 
 
 def wait_for_events(
-    events,
-    timeout=600,
-    sleep_delay=5 * u.second,
-    callback=None,
-):
+    events: Event | list[Event],
+    timeout: float | u.Quantity = 600,
+    sleep_delay: float | u.Quantity = 5 * u.second,
+    callback: callable | None = None,
```

Comment:
```suggestion
    callback: Callable | None = None,
```
Should be the `Callable` type, not the `callable` built-in.